cob_bringup/tools/autoinit.launch

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<launch>
+	<node pkg="cob_helper_tools" type="auto_init.py" name="auto_init" output="screen">
+		<rosparam param="components">[base]</rosparam>
```

Comment:
the `components` should be parameterizable in order to be usable for other robots with e.g. components `[base, torso, head]`, i.e. cob4

in msh, we introduced a new [`components.yaml`](https://github.com/ipa320/msh/blob/indigo_dev/msh_robot_config/robots/cob4-7/config/components.yaml) and [use it like so](https://github.com/ipa320/msh/blob/indigo_dev/msh_bringup/launch/application.xml#L59-L64)

we could discuss about moving this into yaml into each robots bringup layer....i.e. from `msh` to `cob_robots`
@ipa-fez @ipa-jba what do you think?

Reply:
AFAIK we only use auto-bringup on the raw3-3 (which I made this file for) but It sounds nice to have it in a generic way.